universal-login-ops/package.json

```diff
@@ -2,16 +2,17 @@
   "name": "@universal-login/ops",
   "version": "0.4.2",
   "description": "",
-  "main": "dist/src/index.js",
+  "main": "dist/cjs/src/index.js",
   "bin": {
     "universal-login": "./bin/universal-login"
   },
   "scripts": {
     "prebuild": "yarn clean",
-    "build": "tsc --build tsconfig.build.json && cpx \"src/contracts/**/*\" dist/src/contracts",
+    "build": "yarn build:cjs",
+    "build:cjs": "tsc --outDir dist/cjs --p tsconfig.build.json && cpx \"src/contracts/**/*\" dist/cjs/src/contracts",
```

Comment:
#ops-jsons

Reply:
remove cpx and move `MockDai.json` and `Token.json` from `universal-login-ops/src/contracts` to `universal-login-ops/src/contracts/beta-2`